src/Infrastructure/Persistence/Configuration/MemberConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Codidact.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Codidact.Infrastructure.Persistence.Configuration
+{
+    /// <summary>
+    /// Mapping configuration for the Member entity
+    /// </summary>
+    public class MemberConfiguration : IEntityTypeConfiguration<Member>
+    {
+        public void Configure(EntityTypeBuilder<Member> builder)
+        {
+            builder.Property(t => t.DisplayName)
+                .HasMaxLength(100)
+                .IsRequired();
+
+            builder.Property(t => t.Email)
+                .HasMaxLength(255)
```

Comment:
[Email addresses may be up to 320 characters long](https://stackoverflow.com/questions/386294/what-is-the-maximum-length-of-a-valid-email-address).

Reply:
Fixed in b8ef25892bf8085ea0b69ce3b4e4aa29f144c86c